src/sidebar/util/version-data.js

```diff
@@ -1,30 +1,25 @@
 /**
- * An object representing user info
- *
- * @typedef {Object} UserInfo
- * @property {string=} userid
- * @property {string=} displayName
+ * @typedef {import('../components/user-menu').AuthState} AuthState
```

Comment:
This looked like a partial `AuthState` type, I felt it was reasonable to just make it the same thing.